src/web/app/pages-instructor/instructor-session-individual-extension-page/individual-extension-date-modal/individual-extension-date-modal.component.scss

```diff
@@ -0,0 +1,3 @@
+.text-danger {
+  margin: 0;
+}
```

Comment:
Let's add `m-0` to the class instead. I don't think we should be overriding the bootstrap css classes unless necessary.

Reply:
![image](https://user-images.githubusercontent.com/10182564/161887465-2d97a103-b296-4f78-9eae-d92f04ca2b01.png)

Bootstrap's implementation of m-0 is the same, would there be a difference?